google/cloud/bigquery/schema.py

```diff
@@ -291,3 +318,72 @@ def _to_schema_fields(schema):
         field if isinstance(field, SchemaField) else SchemaField.from_api_repr(field)
         for field in schema
     ]
+
+class PolicyTagList(object):
+    """Define Policy Tags for a column.
+
+    Args:
+        names (Union[List[str], None]): list of policy tags to associate with
+            the column.
+    """
+
+    def __init__(self, names=None):
+        self._properties = {}
+        if names is not None:
+            self.names = names
```

Comment:
Is there a reason to have the `is not None` check here, too? Asking because it's already included in the property setter's logic and is partially duplicated here.

Reply:
Done